WebRoot/css/zm.css

```diff
@@ -4398,18 +4398,21 @@ HTML>BODY .appt-selected .appt_allday_body
 
 .ZmShortcutList {
 	padding: 1rem;
-	-moz-columns: 3;
-	-webkit-columns: 3;
-	columns: 3;
+	-moz-columns: 2;
```

Comment:
can we create a property in skin.properties for this, so we will be able to check browser type and apply specific prefixed version, check DisableSelection for example